interchange/DeviceResources.capnp

```diff
@@ -590,4 +591,47 @@ struct Device {
     # Which sites have LUT BELs?
     lutElements @1 : List(LutElements);
   }
+
+  enum ParameterFormat {
```

Comment:
for the sake of completeness, Lattice also use C-style binary strings like https://github.com/YosysHQ/yosys/blob/master/techlibs/ecp5/cells_sim.v#L624-L625

Reply:
Sure

Reply:
Fixed